tests/delete_regress/tests.py

```diff
@@ -255,9 +255,9 @@ def test_delete_proxy_pair(self):
         self.assertEqual(len(FooFileProxy.objects.all()), 0)
 
     def test_19187_values(self):
-        with self.assertRaises(TypeError):
+        with self.assertRaises(AssertionError):
```

Comment:
I thought of that, I tried to blend in with the surrounding code since the tests were similar.